assignments/prototypes.js

```diff
@@ -32,7 +58,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
+
+function Humanoid(humanoidAttributes) {
+  CharacterStats.call(this,humanoidAttributes);
+  this.team = humanoidAttributes.team;
+  this.weapons = humanoidAttributes.weapons;
+  this.language = humanoidAttributes.language;
+}
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+Humanoid.prototype.greet = function() {
+  return `${this.name} offers a greeting in ${this.language}`;
+}
```

Comment:
Very well executed setting up the constructors and prototypes of these things, including inheritance